modules/admin-ui/src/main/webapp/scripts/shared/services/deleteSingleSeriesModalService.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+'use strict';
+
+/**
+ * @ngdoc service
+ * @name adminNg.modal.Modal
+ * @description
+ * Provides a service for displaying the delete single series confirmation dialog.
+ * partial.
+ */
+angular.module('adminNg.services.modal')
+.factory('DeleteSingleSeriesModal', ['$location', 'Modal', 'SeriesHasEventsResource', 'SeriesOptionsResource', function ($location, Modal, SeriesHasEventsResource, SeriesOptionsResource) {
+    var DeleteSingleSeriesModal = function () {
+        var me = this;
+
+        /**
+         * @ngdoc function
+         * @name Modal.show
+         * @methodOf adminNg.modal.Modal
+         * @description
+         * Displays a modal and its overlay.
+         *
+         * Loads markup via AJAX from 'partials/modals/{{ modalId }}.html'.
+         *
+         * @param {string} modalId Identifier for the modal.
+         * @param {string} callback Name of the function to call when confirmed
+         * @param {string} object Hash for the success callback function
+         */
+        this.show = function (modalId, callback, object) {
+            Modal.show(modalId);
+            me.$scope = Modal.$scope;
+
+            me.$scope.confirm  = me.confirm;
+            me.$scope.callback = callback;
+            me.$scope.object   = object;
+            me.$scope.name = "undefined";
+            me.$scope.type = "UNKNOWN";
+            if (!angular.isUndefined(object)) {
+                me.$scope.id = object.id;
+                if (object.title) {
+                    me.$scope.name = object.title;
```

Comment:
Why switch from `title` to `name`?

Reply:
Made sense to me at that point in time. I fear the overhead changing this now.